.gitignore

```diff
@@ -24,7 +24,7 @@ coverage
 public/bundle.js
 
 # Environment variables
-dbpass.js
+.env
```

Comment:
Have you tested this on your AWS instance yet? Where is it getting the environment variables from if you're gitignoring (rightfully, don't commit this, lol) the .env file... Are you successfully passing environment variables from the elastic beanstalk dashboard?